### ISSUE TITLE
Design issues 

### ISSUE THREAD

**Issue by Sajadkim** [microsoft/SmartKC-A-Smartphone-based-Corneal-Topographer#15]
Is it possible to modify the internal design of the Placido base by increasing the thickness of the columns of the internal cone and increasing its thickness a little, because there is no MJF printing in my country, only resin and flint without affecting the code and the program?

```[tasklist]
### Tasks
```

**Comment by mohja1** [comment]
You can definitely give it a shot.
One of the problems that we faced with resin prints was that it was tricky to remove the "supports". It resulted in bad edges obstructing light or breakage of a few mires.

**Comment by sidgairo18** [comment]
<img width="448" alt="Screenshot 2024-03-20 at 09 54 44" src="https://github.com/microsoft/SmartKC-A-Smartphone-based-Corneal-Topographer/assets/9148202/ad242f02-d9cd-42ad-ad40-47eb9256dd0f">

Yes, I agree with Mohit, you can give it a shot. Another factor to be careful about is, when you increase the width/thickness of the columns in the internal cone, this will lead to wider gaps in the mire image as well (see horizontal empty spaces in attached image), which then might lead to inaccurate reconstructions.

Have fun building on SmartKC :)

**Comment by Sajadkim** [comment]
Thank you very much sir for the reply. I encountered problems when printing the Placido base using resin or filament. The inner cone breaks, so I will order it to be printed from outside the country. My question is whether the design, if printed using MJF, will withstand and be strong. This is the characteristics of the piece and the material is PA12 nylon.
![IMG_4436](https://github.com/microsoft/SmartKC-A-Smartphone-based-Corneal-Topographer/assets/164097522/ea5a29d3-20ed-4f1d-9cda-08908a346714)

**Comment by sidgairo18** [comment]
MJF printed Placido is reasonably strong.

about the material @mohja1 would be able to confirm.

Thanks!

**Comment by mohja1** [comment]
Yes, it is strong. We used MJF PA12 in Natural Grey color with Matte finish.
